docs-developer/CHANGELOG.md

```diff
@@ -0,0 +1,79 @@
+# Processed and Gecko Profile Format Changelog
+
+This file documents all changes in the profiler gecko and processed formats.
+
+Note that this is not an extensive list. Processed profile format upgraders can be found in [processed-profile-versioning.js](../src/profile-logic/processed-profile-versioning.js) and gecko profile format upgraders can be found in [gecko-profile-versioning.js](../src/profile-logic/gecko-profile-versioning.js). Please refer to them for older upgraders or for exact implementations.
```

Comment:
did you mean "exhaustive" rather than "extensive"?

Reply:
Oops yeah "exhaustive" was the word I was looking for :)